### DISCUSSION TITLE
【求助】尼尼微打不过啊，黑简打不过啊

### DISCUSSION THREAD

**Discussion by CyccyYan** [share121/inter-knot#3068] [其他]
这™究竟谁能打过啊<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by Pheathon** [comment]
这个委托我接了（bushi）

**Comment by zky-ui** [comment]
试试练一个本，四十五级，两个大防御盘子就够

**Comment by fairy-fire** [comment]
我现在也打不过，尼尼微难度九我都打不过打算，等到养毕业了再打<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/8b9838175cb6312df34c32648a8e6649_1050729359995605746.png" alt="景元 吸猫" width="75" >

**Comment by qqqqqq77777** [comment]
我直♂接不打<img src="https://bbs-static.miyoushe.com/static/2023/11/16/e03be25a7b2bc86f2b1909dddb6ca915_6686505940047795068.png" alt="安东-元气满满" width="75" >

**Comment by SherlockAmo** [comment]
没事，一开始都这样，熟练之后就很容易，成就感还是满满的
![1724076741839](https://github.com/user-attachments/assets/cc4887b5-d94d-487e-bc51-65879891a034)

**Comment by Pray-Fay** [comment]
![F424417CE504DA8EC78594F209322718](https://github.com/user-attachments/assets/ee68ea5a-5056-4ee4-b576-725869ebfc5b)
可以用鲨鱼的巡游<img src="https://img-static.mihoyo.com/communityweb/upload/2b1154fdc2718859060d789fbfcb8224.png" alt="米游姬 好耶" width="75" >青衣也可以过

**Reply by Pray-Fay** [reply]
刚把青衣拉到50级，试过单通，录了十来次视频最好的一次是打到黑简剩大概四分之一血，理论应该可以过

**Comment by ljy6776** [comment]
尼尼微多买点鸣徽就行，黑简有个轮椅打法，用大本一直按格挡反击，我本来差点翻车的，结果想起了我3号位那个6个双爆驱动，主词条防御防御回能都双爆面板比我11号高的大本，然后直接干过去了
